File: app/src/main/java/hk/collaction/contentfarmblocker/ui/activity/MainActivity.java
package hk.collaction.contentfarmblocker.ui.activity;

import android.content.res.Configuration;
import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.appcompat.app.ActionBar;
import androidx.appcompat.widget.Toolbar;

import butterknife.BindView;
import butterknife.ButterKnife;
import hk.collaction.contentfarmblocker.C;
import hk.collaction.contentfarmblocker.R;
import hk.collaction.contentfarmblocker.ui.fragment.MainFragment;

public class MainActivity extends BaseActivity {

	@BindView(R.id.toolbar)
	Toolbar toolbar;

	@Override
	public void onConfigurationChanged(@NonNull Configuration newConfig) {
		super.onConfigurationChanged(newConfig);
		C.detectLanguage(mContext);
		ActionBar ab = initActionBar(getSupportActionBar(), R.string.title_activity_main);
		ab.setDisplayHomeAsUpEnabled(false);
		ab.setHomeButtonEnabled(false);
	}

	@Override
	public void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_container);
		ButterKnife.bind(this);
		setSupportActionBar(toolbar);

		ActionBar ab = initActionBar(getSupportActionBar(), R.string.title_activity_main);
		ab.setDisplayHomeAsUpEnabled(false);
		ab.setHomeButtonEnabled(false);

		boolean isNoBrowser = false;
		if (getIntent() != null) {
			isNoBrowser = getIntent().getBooleanExtra("no_browser", false);
		}

		Fragment fragment = MainFragment.newInstance(isNoBrowser);
		FragmentManager fragmentManager = getSupportFragmentManager();
		fragmentManager.beginTransaction()
				.replace(R.id.container, fragment)
				.commit();
	}
}

File: app/src/main/java/hk/collaction/contentfarmblocker/ui/activity/MainActivity.kt
package hk.collaction.contentfarmblocker.ui.activity

import android.os.Bundle
import hk.collaction.contentfarmblocker.R
import hk.collaction.contentfarmblocker.helper.UtilHelper
import hk.collaction.contentfarmblocker.ui.fragment.MainFragment

class MainActivity : BaseFragmentActivity() {
    override var titleId: Int? = R.string.title_activity_main

    override fun onCreate(savedInstanceState: Bundle?) {
        setTheme(R.style.AppTheme)
        val isNoBrowser: Boolean = intent.getBooleanExtra("no_browser", false)
        fragment = MainFragment.newInstance(isNoBrowser)
        super.onCreate(savedInstanceState)
        supportActionBar?.setHomeButtonEnabled(false)
        supportActionBar?.setDisplayHomeAsUpEnabled(false)
    }
}